data/tekton-pipelines/windows-customize-pipeline.yaml

```diff
@@ -54,8 +54,6 @@ spec:
             apiVersion: cdi.kubevirt.io/v1beta1
             kind: DataVolume
             metadata:
-              annotations:
```

Comment:
You should keep this annotation on the `spec.source.pvc` DVs
It's only broken not honored in `spec.source.blank` Block DVs

Reply:
Ofc if youre creating a VM out of those DVs then the annotation just doesn't make much sense
so up to you